lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -952,6 +952,17 @@ def _create_deployment_parameters(
             parameters=parameters,
         )
 
+        # dump arm_template and arm_parameters to file
+        dump_path = os.path.join(os.getcwd(), "runtime/runs/arm_dump")
+        try:
+            os.mkdir(dump_path)
+        except OSError:
+            pass
+        dump_file = open(f"runtime/runs/arm_dump/{environment.name}.txt", "w")
+        dump_file.write("arm_template\n" + json.dumps(template))
```

Comment:
dump the original json format is better for view. So dump two files, one for template, another for parameters.